packages/proto-loader/src/index.ts

```diff
@@ -395,6 +395,23 @@ export function loadSync(
   return createPackageDefinition(root, options!);
 }
 
+export function fromJSON(
+  json: Protobuf.INamespace,
+  root?: Protobuf.Root
+): PackageDefinition {
+  const options: Options = json.options || {};
```

Comment:
Does `json.options` actually contain the same options that are used in `createPackageDefinition`?